tools/conformance.py

```diff
@@ -728,12 +742,15 @@ def __init__(self, path, browser, host_os, target_os, android_device=None, debug
                     service_args = []
                 self.driver = selenium.webdriver.Chrome(executable_path=self.path, chrome_options=chrome_options, service_args=service_args)
             elif browser.is_safari():
-                Util.not_implemented()
+                capabilities = DesiredCapabilities.SAFARI.copy()
+                if tools:
+                    capabilities['automaticInspection'] = True
+                self.path = browser.path
+                self.driver = selenium.webdriver.safari.webdriver.WebDriver(desired_capabilities=capabilities, executable_path=self.path)
```

Comment:
browser.path should be assigned to capabilities['binary'], and self.path is actually webdriver path.